File: Raven.Abstractions/FileSystem/SourceSynchronizationInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Raven.Abstractions.FileSystem
{
    public class SourceSynchronizationInformation
    {
        public Guid LastSourceFileEtag { get; set; }
        public string SourceServerUrl { get; set; }
        public Guid DestinationServerId { get; set; }

        public override string ToString()
        {
            return string.Format("LastSourceFileEtag: {0}", LastSourceFileEtag);
        }
    }
}

File: src/Raven.Studio/typescript/viewmodels/filesystem/status/sourceSynchronizationInformation.ts
class sourceSynchronizationInformation {
    LastSourceFileEtag = ko.observable<string>();
    SourceServerUrl = ko.observable<string>();

    constructor(lastSynchronizedEtag: string, sourceServerUrl: string) {
        this.LastSourceFileEtag(lastSynchronizedEtag);
        this.SourceServerUrl(sourceServerUrl);
    }
}

export = sourceSynchronizationInformation;